### ISSUE TITLE
BaseLayerMatching to Browse Imagery not working.

### ISSUE THREAD

**Issue by MLGTimMLG** [DigitalGlobe/gbdxtools#457]
When trying to:
image.plot(blm=True, blm_source="browse"):
ValueError: Invalid metadata returned by metadata service. Cannot create BrowseImage

**Comment by drwelby** [comment]
What class is your `image` object?

**Comment by chelm** [comment]
@MLGTimMLG Providing a bit more code here would help us and others out. We'd like to be able to look at the issue and reproduce the problem with out guessing. Thanks for your issues!

**Comment by MLGTimMLG** [comment]
from gbdxtools.task import env
from gbdxtools import CatalogImage
%matplotlib inline

catalog_id = env.inputs.get('catalog_id', 'Doesnt matter. It doesn't work for the most Images i tried')
bbox = env.inputs.get('bbox', 'some bbox')

image = CatalogImage(catalog_id, pansharpen=True, band_type="MS", bbox=map(float, bbox.split(",")))
image.plot(w=10, h=10, blm=True, blm_source="browse")

**Comment by chelm** [comment]
@MLGTimMLG What version of gbdxtools do you have?

**Comment by MLGTimMLG** [comment]
How do I look that up? (I use GBDX Notebooks)

**Comment by chelm** [comment]
`!conda list | grep gbdxtools`

**Comment by MLGTimMLG** [comment]
0.15.11

**Comment by chelm** [comment]
Ok, there'll be an update to GBDX Notebooks with the latest gbdxtools today. Will notify you when that happens and you'll need to reset your notebook container (upper right in the user drop down). That will get the latest version into your NB environment. Will let you know.

**Comment by chelm** [comment]
<img width="1123" alt="untitled129" src="https://user-images.githubusercontent.com/351164/44478442-f07e0d80-a5fa-11e8-95c6-78967b2f861a.png">
Verified working in 0.15.12

**Comment by MLGTimMLG** [comment]
@chelm Thank you

**Comment by chelm** [comment]
Fixed in latest gbdxtools/notebooks v 0.15.12

```python
from gbdxtools.task import env
from gbdxtools import CatalogImage
%matplotlib inline

catalog_id = env.inputs.get('catalog_id', 'e9d158c6-2af0-481d-a1f0-d82b7234db33-inv')

image = CatalogImage(catalog_id, pansharpen=True, band_type="MS")
image = image.randwindow((256,256))
image.plot(w=10, h=10, blm=True, blm_source="browse")
```
